packages/core/src/modules/credentials/protocol/v2/__tests__/V2CredentialServiceCred.test.ts

```diff
@@ -790,6 +803,16 @@ describe('CredentialService', () => {
 
       await credentialService.delete(credentialRecord)
 
+      expect(deleteCredentialMock).toHaveBeenNthCalledWith(1, credentialRecord.credentials[0].credentialRecordId)
+    })
+    it('deleteAssociatedDidCommMessages should default to true', async () => {
+      const deleteCredentialMock = mockFunction(indyCredentialFormatService.deleteCredentialById)
+
+      const credentialRecord = mockCredentialRecord()
+      mockFunction(credentialRepository.getById).mockResolvedValue(credentialRecord)
+
+      await credentialService.delete(credentialRecord)
+
       expect(deleteCredentialMock).toHaveBeenNthCalledWith(1, credentialRecord.credentials[0].credentialRecordId)
```

Comment:
This doesn't check if the didcomm messages are deleted. We can check this using `expect(didCommMessageRepository.delete).toHaveBeenCalledTimes(3)` and then checking if it has called it for each of the record returned by findByQuery